js_machine_front/src/routes/News.tsx

```diff
@@ -1,16 +1,48 @@
-import React, { memo } from 'react';
+import React, { PureComponent } from 'react';
+import DigestView from 'components/DigestView/DigestView';
 
 const sectionStyle = {
   backgroundColor: 'grey',
+  paddingTop: 100,
 };
 
-export const News: React.FC = memo(() => {
-  return (
-    <div style={ sectionStyle }>
-      <h1>News</h1>
-      <div>
-        <h3>Page!</h3>
+interface State {
+  openDigest: boolean;
+}
+
+export class News extends PureComponent<{}, State> {
+
+  constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      openDigest: false,
+    };
+  }
+
+  openDigest = () => {
+    this.setState({ openDigest: true });
+  }
+
+  closeDigest = () => {
+    this.setState({ openDigest: false });
+  }
+
+  exitKeyPressHandler = (e: any) => {

```

Comment:
don't use any, try to find correct type
the same for other places in code

Reply:
okey

Reply:
hey, what kind of event is that? i thought it was KeyboardEvent but it gives me some error that says KeyboardEvent is missing keycode etc all properties )